### ISSUE TITLE
Update Empty state guidelines to show new illustrations

### ISSUE THREAD

**Issue by mmenestr** [patternfly/patternfly-org#3273]
Changing Empty states to use illustrations instead of icons.
Would need to update all the images to match these new illustrations, as well as the icon table for the different use cases. 

ON HOLD: Waiting on this to become the official thing we want to advertise. Could be on hold until the big release/PF5 in June.

**Comment by mmenestr** [comment]
Would refer to this issue for the latest - https://github.com/patternfly/patternfly-design/issues/1284. For now, these illustrations were not brought into our Empty states, but there might be something related to this done in the future with PF6 and the potential merge with the Red Hat icon library.
